USING_DOCKER.md

```diff
@@ -0,0 +1,123 @@
+# Using the Dockerfile
```

Comment:
I would still vote to have this in Developing, but we can try it out here for a bit if you prefer. My logic is that people will be making a decision to develop and then thinking how they do it, rather than the other way around. And the root of the project is already quite busy, I'm planning to try and cut those files down as much as possible. (It seems like a small thing! But the more files are there, the less we can see anything about the project when viewing the GH project page!)

We should at least add a link between them if we do keep it

Reply:
These are very real concerns from a global project standpoint. I'm quite sympathetic to your concern about a busy root directory.

At this stage, I'd counsel keeping the USING_DOCKER.md file separate. It provides a lot of motivation and hand-holding specific to the use of Docker. 

If it becomes an important tool for development, then it'll be clear how important it would be to merge the DEVELOPMENT.md and USING_DOCKER.mg files.

Reply:
Completely fine, let's see how it goes for a few months